MYabrv.bib

```diff
@@ -358,6 +358,7 @@ @String{NJC
 @String{NRL = "Naval Research Logistics"}
 @String{NeurIPS = "Advances in Neural Information Processing Systems (NeurIPS)"}
 @String{OSR = "ACM SIGOPS Operating Systems Review"}
+@String{PACMHCI = "Proceedings of the ACM on Human Computer Interaction (PACMHCI)"}
```

Comment:
IMO this should be spelled Human-Computer Interaction.
their own website seems to handel this very inconsistently: https://dl.acm.org/journal/pacmhci
what do we want to do?

Reply:
This is unfortunate. :( I vote no `-` because then it is at least consistent within BibTags (except for `VLHCC`). I do not have a strong opinion though.

Reply:
works for me.